### PR TITLE
Restore line coverage

### PR DESCRIPTION
**Pull request by martiniil**
brings back line coverage to 100%

### CODE REVIEW

File: pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.test [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0"?>
```
Comment (agutenkunst) [inline pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.test:1]: Please don't do a rename to a file you completely change

File: pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.cpp [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    FAIL() << "Robot model could not be loaded.";
+  }
+}
+
+/**
+ * @brief Check if hasSolver() can be called successfully for the manipulator group.
+ */
+TEST_F(GetSolverTipFrameIntegrationTest, TestHasSolverManipulator)
+{
+  try
```
Comment (agutenkunst) [inline pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.cpp]: Please state the intention behind catching everything...

**Review by agutenkunst — CHANGES_REQUESTED** [review]

File: pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.cpp (hunk 2 above)
Comment (martiniil) [reply pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.cpp]: Is there a drawback?

Comment (agutenkunst) [reply pilz_trajectory_generation/test/integrationtest_get_solver_tip_frame.cpp]: if there is no benefit: KISS

**Review by agutenkunst — APPROVED** [review]

**Pull request merged**